Standup-Meetings-Getting-Started.md

```diff
@@ -0,0 +1,35 @@
+# Standup Meetings - Getting Started
+
+Stand-up Meeting is one of the two productivity-enhancing features of Probot. To gain maximum benefits, learn how the app works and ways to potentially increase the productivity of your team using this slackbot.
+
+### Create a Slack Account
+To use Probot, a Slack account is a must. All user profiles should be linked to the existing workspace profile on Slack. If your workspace profile is yet to be created, we recommend you to first 'Create Workspace'. Once it is done, create your individual Slack account and add to your workspace.
+
+### Add Probot to Slack
+Next step is adding Probot to Slack. Visit https://probot.ninja and hit the button which says “Add to Slack”. When you do this, Probot will be linked to your Slack account and be ready to use.  
+
+### Create a Channel
+Create a dedicated Slack channel where daily scrums would take place and assign a name. If you want to conduct daily stand-ups for more than one team, create multiple channels and follow the same procedure.
+
+### Add Probot to Channel
+Add Probot to the daily stand-up channel. To do this, you may either type ‘Probot Ninja’ in the ‘Send Invites To’ section while creating a channel or add the app later.
+
+### Add People
+To add members to the scrum channels, make sure they have existing Slack accounts linked to the workspace. If you want to add people when you are creating a channel, write their names in 'Send Invites To" section. You can always add more members from 'Invite More People' option on Slack.
+
+## Customize Scrum Questions
+By default, Probot will ask a pre-defined set of questions. Those are:
+
+•	What did you do yesterday?
+
+•	What will you do today?
+
+•	Any roadblock.
+
+If you want to add a different set of questions, overwrite the existing questions or ‘Add New Question’.
```

Comment:
We should hyperlink directly to Customize-Standup section from here, rather than telling users how to customize scrum questions

Reply:
I could not find any link that directly takes the user to edit the questions. I request you to please guide me what words/sentences to be removed and what are to be added in this case. Or provide with a link.